spec/models/post_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { create(:user) }
+
+  it 'is valid with valid attributes' do
+    post = build(:post, author: user)
+    expect(post).to be_valid
+  end
+
+  it 'is not valid without a title' do
+    post = build(:post, title: nil, author: user)
+    expect(post).not_to be_valid
+  end
+
+  it 'is not valid with a title exceeding 250 characters' do
+    post = build(:post, title: 'a' * 251, author: user)
+    expect(post).not_to be_valid
+  end
+
+  it 'is not valid without text' do
+    post = build(:post, text: nil, author: user)
+    expect(post).not_to be_valid
+  end
+
+  it 'updates the user posts counter after saving' do
+    # post = create(:post, author: user)
```

Comment:
Kindly ensure to take all commented pieces of code not in use as they make your codebase messy and give it a less professional look😉.